st2actions/st2actions/cmd/actionrunner.py

```diff
@@ -19,9 +20,22 @@
     time=True)
 
 
+def _setup_sigterm_handler():
```

Comment:
I'd prefer a base class that individual classes can inherit from - BaseProcess or something like that. I haven't thought about how it would fit our existing code.